app/services/courtfinder_api.rb

```diff
@@ -0,0 +1,43 @@
+require 'json'
+require 'open-uri'
+
+class CourtfinderAPI
```

Comment:
Can you put this service inside the namespace `C100App`? That way, mutant will get to test it automatically (anything inside the `C100App` namespace gets mutant-tested).

Reply:
done